frontend/src/scenes/funnels/FunnelCanvasLabel.tsx

```diff
@@ -66,6 +68,24 @@ export function FunnelCanvasLabel(): JSX.Element | null {
                   </>,
               ]
             : []),
+
+        ...(supportsCreatingExperiment
+            ? [
+                  <LemonButton
+                      key="run-experiment"
+                      icon={<IconTestTube />}
+                      type="secondary"
+                      data-attr="create-experiment-from-insight"
+                      size="xsmall"
+                      to={urls.experiment('new', {
+                          insight: insight.short_id ?? undefined,
+                          name: (insight.name || insight.derived_name) ?? undefined,
+                      })}
+                  >
+                      Run Experiment
```

Comment:
We use title case in our copy:
```suggestion
                      Run experiment
```